### DISCUSSION TITLE
Any interest in coupling phase-field and electrochemistry?

### DISCUSSION THREAD

**Discussion by ZhigangPu** [idaholab/moose#19344] [Q&A Modules: Phase field] [answered]
Dear,

Phase-field method are growing popular in studying the interface between electrode and electrolyte in batteries, e.g. Lithium metal batteries. It's vital to develop new batteries to solve energy issues or other systems containing electrochemistry interface for practical applications. 

Here are some milestone papers trying to apply phase-field methods into electrochemistry:

1. J.E. Guyer, W.J. Boettinger, J.A. Warren, G.B. McFadden, Phys. Rev. E 69 (2004) 021603.
2. J.E. Guyer, W.J. Boettinger, J.A. Warren, G.B. McFadden, Phys. Rev. E 69 (2004) 021604.
3. Y. Shibuta, Y. Okajima, T. Suzuki, Sci. Technol. Adv. Mater. 8 (2007) 511e518. [25] L. Liang, Y. Qi, F. Xue, S. Bhattacharya, S.J. Harris, L.-Q. Chen, Phys. Rev. E 86 (2012) 051609.
4. D.R. Ely, A. Jana, R.E. García, J. Power Sources 272 (2014) 581e594. [27] M.Z. Bazant, Acc. Chem. Res. 46 (2013) 1144e1160. [28]
5. Chen, L., Zhang, H. W., Liang, L. Y., Liu, Z., Qi, Y., Lu, P., Chen, J., & Chen, L. Q. (2015). Modulation of dendritic patterns during electrodeposition: A nonlinear phase-field model. Journal of Power Sources, 300, 376–385. 
6. Cogswell, D. A. (2015). Quantitative phase-field modeling of dendritic electrodeposition. Physical Review E - Statistical, Nonlinear, and Soft Matter Physics, 92(1), 1–5. https://doi.org/10.1103/PhysRevE.92.011301

I wonder is there someone interested in writing such a module in MOOSE. If so, I guess we can cooperate to write such module.

**Comment by laagesen** [comment, marked as answer]
There has been some electrochemical phase-field models implemented in MOOSE in the past, but there is no specific application for them. However you might be able to find kernels already written for the model you are interested in in the phase-field repository. 

https://journals.aps.org/pre/abstract/10.1103/PhysRevE.101.022802

**Reply by ZhigangPu** [reply]
Sorry for late reply, many thanks for the guidance and paper!

zhigang
